qemu/tests/boot_N_M_virtserialports.py

```diff
@@ -21,9 +22,18 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
+    if params.get("start_vm") == 'no':
+        num_bus = params.get_numeric("numberic_bus")
+        for i in range(2, num_bus + 1):
+            serial_name = 'vs%d' % i
+            params['serials'] = '%s %s' % (params.get('serials', ''), serial_name)
+            params['serial_type_%s' % serial_name] = "virtserialport"
+            params['serial_reuse_bus_%s' % serial_name] = "no"
```

Comment:
Please update this, according to the latest solution of :
https://github.com/avocado-framework/avocado-vt/pull/2514

And, update the case configuration for 'boot_with_multiple_virtserialports.N_M' together.

Reply:
sure, thanks and update.